internal/grpc/services/gateway/ocmshareprovider.go

```diff
@@ -56,6 +56,9 @@ func (s *svc) CreateOCMShare(ctx context.Context, req *ocm.CreateOCMShareRequest
 	if err != nil {
 		return nil, errors.Wrap(err, "gateway: error calling CreateOCMShare")
 	}
+	if res.Status.Code != rpc.Code_CODE_OK {
```

Comment:
Can't we use getters?
```suggestion
	if res.GetStatus().GetCode() != rpc.Code_CODE_OK {
```